flink-kubernetes/src/main/java/org/apache/flink/kubernetes/kubeclient/decorators/KerberosMountDecorator.java

```diff
@@ -107,7 +107,7 @@ public FlinkPod decorateFlinkPod(FlinkPod flinkPod) {
                             .withItems(
                                     new KeyToPathBuilder()
                                             .withKey(krb5Conf.getName())
-                                            .withPath(krb5Conf.getName())
+                                            .withPath("krb5.conf")
```

Comment:
nit: it would be good to introduce a constant for it in `org.apache.flink.kubernetes.utils.Constants`.

Reply:
Thanks for your review.  added `org.apache.flink.kubernetes.utils.Constants.KERBEROS_KRB5CONF_FILE`